src/courtroom.cpp

```diff
@@ -72,9 +72,13 @@ Courtroom::Courtroom(AOApplication *p_ao_app) : QMainWindow()
   ui_vp_testimony->setAttribute(Qt::WA_TransparentForMouseEvents);
   ui_vp_wtce = new SplashLayer(this, ao_app);
   ui_vp_wtce->set_play_once(true);
+  ui_vp_wtce->continuous = false;
+  ui_vp_wtce->force_continuous = true;
   ui_vp_wtce->setAttribute(Qt::WA_TransparentForMouseEvents);
   ui_vp_objection = new SplashLayer(this, ao_app);
   ui_vp_objection->set_play_once(true);
+  ui_vp_objection->continuous = false;
+  ui_vp_objection->force_continuous = true;
```

Comment:
Something seems fishy with this but I lack the braincells at the moment to understand my intuition

Reply:
`continuous` is set automatically using a simple file path check (is this the same file we just loaded). if you want to bypass that check, you need to set `force_continuous` to true - if `force_continuous` is true, `start_playback()` will blindly use whatever the value of `continuous` already is and not perform any check at all

Reply:
the reason this smells fishy is because it is a terrible hack, but a necessary hack as far as I can tell